Automattic/wp-calypso
# Add/jitm for agencies in backup cloud

## Proposed Changes

This PR adds a JITM-like banner to the Jetpack Backup Cloud page that shows up only to Agencies/Partners that links to a survey for them to fill out

NOTE: I also made a new directory for all the banners in `my-sites/backup`, so we'll make sure they all still work as well

P2: pbNhbs-5cV-p2
Design convo: p1680716915761929-slack-C0D96691V

## Testing Instructions

1. Checkout this branch
2. Start Calypso Green via `yarn start-jetpack-cloud`
3. If you don't have one already, purchase a Jetpack Backup license and attach it to a site (I used jurassic ninja). Do not enter the server credentials
4. Go to http://jetpack.cloud.localhost:3000/backup/{your-site} 
5. Ensure the server credentials banner still shows up after moving the files to their own folder
![Screenshot 2023-04-05 at 1 26 16 PM](https://user-images.githubusercontent.com/65001528/230184387-230c1261-43d5-401b-a24c-d4db8650d90b.jpg)
6. Make sure the new Agencies banner does not show up yet (unless you're already signed up as a Jetpack Partner)
7. Follow these instructions ( PCYsg-KIj-p2 ) to sign up as a Jetpack Partner. Make sure you choose Partner type of "Agency"
8. Refresh the Jetpack Cloud Backup page and ensure the new banner now shows up
![Screenshot 2023-04-05 at 1 28 36 PM](https://user-images.githubusercontent.com/65001528/230184854-900e5e0e-8931-4051-b4e3-f7fc0bb94195.jpg)
9. Make sure the CTA works and sends them to the correct URL
10. Dismiss the Banner and make sure it doesn't come back on refresh
![image](https://user-images.githubusercontent.com/65001528/230185032-1aa2d506-99eb-455f-94cf-6eb3bf485668.png)

NOTE: If you dismiss the banner and need it to be visible again, in development, you can hover over `JETPACK-CLOUD-DEV`, then Preferences, and at the very bottom there is a checkbox you can un-check to clear the preference
![Screenshot 2023-04-05 at 4 10 34 PM](https://user-images.githubusercontent.com/65001528/230223177-5db9f8f4-e350-437a-a381-37838b577f07.jpg)

## Pre-merge Checklist
